packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.fuel_message_portal_v3.ts

```diff
@@ -25,10 +25,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     response.hash
   );
 
+  const implementation = receipt?.contractAddress ?? '';
+
+  if (implementation === '')
+    throw new Error(
+      `Upgrade proposement failed for FuelMessagePortalV3 proxy (${contractDeployment.address})`
```

Comment:
proposal*

Reply:
google said it was a legit word lol nvm will update